README.md

```diff
@@ -6,33 +6,33 @@
 [![Image Layers](https://badge.imagelayers.io/erlang:latest.svg)](https://imagelayers.io/?images=erlang:latest 'Show Image Layers at imagelayers.io')
 
-[![Build Status](https://travis-ci.org/c0b/docker-erlang-otp.svg?branch=master)](https://travis-ci.org/c0b/docker-erlang-otp)
+[![Build Status](https://travis-ci.org/erlang/docker-erlang-otp.svg?branch=master)](https://travis-ci.org/erlang/docker-erlang-otp)
 
 This is used as docker base image for Erlang OTP.
-The goal is to provide images for a few last erlang releases (currently 20 / 19 / 18), in close to full feature Erlang OTP, and relatively slim images. Support to 17, R16 and R15 are provided in this repo on a best-effort basis, and not part of official-image effort in docker-library/official-images#1075 .
+The goal is to provide images for a few last erlang releases (currently 21/ 20 / 19 / 18), in close to full feature Erlang OTP, and relatively slim images. Support to 17, R16 and R15 are provided in this repo on a best-effort basis, and not part of official-image effort in docker-library/official-images#1075 .
 
-### use the Erlang 20
+### use the Erlang 21
 
-here is providing the latest Erlang 20 image; you may pull from official-images or build it locally:
+here is providing the latest Erlang 21 image; you may pull from official-images or build it locally:
 
 ```console
-$ docker build -t erlang:20.0 ./20
+$ docker build -t erlang:21.0 ./21
 [...]
-➸ docker run -it --rm erlang:20.0
-Erlang/OTP 20 [erts-9.0] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]
+➸ docker run -it --rm erlang:21.0
+Erlang/OTP 21 [erts-10.0] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]
 
-Eshell V9.0  (abort with ^G)
+Eshell V10.0  (abort with ^G)
 1> erlang:system_info(otp_release).
-"20"
+"21"
 2> os:getenv().
-["PWD=/","REBAR3_VERSION=3.4.1",
+["PWD=/","REBAR3_VERSION=3.5.3",
 "ROOTDIR=/usr/local/lib/erlang","LANG=C.UTF-8",
-"PATH=/usr/local/lib/erlang/erts-9.0/bin:/usr/local/lib/erlang/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+"PATH=/usr/local/lib/erlang/erts-10.0/bin:/usr/local/lib/erlang/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
 "REBAR_VERSION=2.6.4","TERM=xterm",
-"BINDIR=/usr/local/lib/erlang/erts-9.0/bin","PROGNAME=erl",
-"EMU=beam","OTP_VERSION=20.0","HOME=/root",
+"BINDIR=/usr/local/lib/erlang/erts-10.0/bin","PROGNAME=erl",
+"EMU=beam","OTP_VERSION=21.0","HOME=/root",
 "HOSTNAME=9b1e7f4d7206"]
-3> 'hello_юникод_世界'.                                   % Erlang20 now support unicode in atom
+3> 'hello_юникод_世界'.                                   % Erlang21 now support unicode in atom
 'hello_юникод_世界'
 4> io:format("~tp~n", [{'hello_юникод', <<"Hello, 世界; юникод"/utf8>>, "Hello, 世界; юникод"}]).
 {'hello_юникод',<<"Hello, 世界; юникод"/utf8>>,"Hello, 世界; юникод"}
```

Comment:
usually with new major version, I'd like to use the README to demo case one piece of brand new feature, like the Unicode atom in Erlang 20;  so for Erlang 21, what do you think is a prominent new feature only available in 21 ?

from http://erlang.org/pipermail/erlang-questions/2018-June/095691.html how do you think the `catch C:R:Stacktrace` syntax ? like

```erlang
1> try 1/0 catch C:R:Stacktrace -> io:format("caught: ~tp", [{C,R,Stacktrace}]) end.
caught: {error,badarith,
               [{erlang,'/',[1,0],[]},
                {erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,681}]},
                {erl_eval,try_clauses,8,[{file,"erl_eval.erl"},{line,911}]},
                {shell,exprs,7,[{file,"shell.erl"},{line,686}]},
                {shell,eval_exprs,7,[{file,"shell.erl"},{line,642}]},
                {shell,eval_loop,3,[{file,"shell.erl"},{line,627}]}]}ok
```

Reply:
If I get a vote, I'd say logger is the most prominent new feature.